digital-twins/jsonld-contexts/digital_twins_contexts.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_attributes.jsonld",
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_entity_types.jsonld",
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_relationships.jsonld",
```

Comment:
don't know what to think about the separation into different files (and btw, a relationship is an attribute)

Reply:
Ok maybe it isn't really necessary, I'll regroup them